_posts/2018-07-17-client-openings.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+author: chris
+title: Let us help!
+image: 'posts/diagram.jpg'
+---
+
+Apsis Labs is about to wrap up our largest contract yet, and has openings for new clients! [Contact us](mailto:contact@apsis.io) to tell us about your project.
```

Comment:
Awesome post! I'm excited to get this out, and add a link to it from homepage copy.